packages/suite/src/views/suite/SwitchDevice/DeviceItem/EjectConfirmation.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { useDispatch } from 'src/hooks/suite';
+import { H3, Button, Text } from '@trezor/components';
+import { Translation } from 'src/components/suite';
+import { deviceActions } from '@suite-common/wallet-core';
+import { analytics, EventType } from '@trezor/suite-analytics';
+import { AcquiredDevice } from '@suite-common/suite-types';
+import { MouseEventHandler } from 'react';
+import { spacingsPx } from '@trezor/theme';
+
+type EjectConfirmationProps = {
+    instance: AcquiredDevice;
+    onCancel: MouseEventHandler<HTMLButtonElement> | undefined;
+    onClick: MouseEventHandler<HTMLDivElement> | undefined;
+};
```

Comment:
For props, we typically use `interface` and put it right above the component.

Reply:
Is there any reason for using interface?

Reply:
No, just to be consistent.